docs/AboutUs.md

```diff
@@ -17,7 +17,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[github](https://github.com/johndoe)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Project Advisor
+- Role: Project Advisor
```

Comment:
Sorry, I don't understand why you changed all the * to - for bullet points? Both styles should work for bullet points in GFMD.

Reply:
Oh, it was from my auto-formatter, which was used to format the table in the developer guide. Changed to `*` to be consistent with the rest for now. Maybe we could standardize which to use in another pull request.